src/renderer/EqualizerEnablerSwitch.tsx

```diff
@@ -0,0 +1,50 @@
+import { ErrorDescription } from 'common/errors';
+import { useCallback, useContext, useState } from 'react';
+import {
+  disableEqualizer,
+  enableEqualizer,
+  getEqualizerStatus,
+} from './equalizerApi';
+import { PeaceFoundContext } from './PeaceFoundContext';
+import Switch from './Switch';
+
+interface IEqualizerEnablerSwitchProps {
+  id: string;
+}
+
+export default function EqualizerEnablerSwitch({
+  id,
+}: IEqualizerEnablerSwitchProps) {
+  const { setPeaceError } = useContext(PeaceFoundContext);
+  const [equalizerEnabled, setEqualizerEnabled] = useState<boolean>(false);
+
+  const equalizerEnablerOnLoad = useCallback(async () => {
+    try {
+      setEqualizerEnabled(await getEqualizerStatus());
+    } catch (e) {
+      setPeaceError(e as ErrorDescription);
+    }
+  }, [setPeaceError]);
+
+  const handleToggleEqualizer = useCallback(async () => {
+    try {
+      if (equalizerEnabled) {
+        await disableEqualizer();
+      } else {
+        await enableEqualizer();
+      }
+      setEqualizerEnabled(!equalizerEnabled);
+    } catch (e) {
+      setPeaceError(e as ErrorDescription);
+    }
+  }, [equalizerEnabled, setPeaceError]);
+
+  return (
+    <Switch
+      isOn={equalizerEnabled}
+      handleToggle={handleToggleEqualizer}
+      id={id}
```

Comment:
does this id need to be passed through two props? it's only really used so that the label's `htmlFor` can be associated to the input of the switch, right?

Reply:
hmmm. i guess it depends whether we make use multiple equalizer enabler switch buttons. the id has to be unique.

Reply:
Right ok that makes sense! Hm, that's gonna need addressing in the `Slider` component too. Maybe we should get some sort of guid library to generate unique ids across the app 🤔. We can tackle that when we get to setting up frequency bands tho.